docs/troubleshoot/ze/ze-known-limits.md

```diff
@@ -0,0 +1,26 @@
+# Known Zowe Explorer limits
+
+### Mismatched credentials when using Zowe Explorer and Zowe CLI
+
+#### Limitation
+
+Mismatching credentials can potentially lock you out from using Zowe CLI and Zowe Explorer in Visual Studio Code with either Windows Subsystem for Linux (WSL) or Remote Secure Shell (SSH).
+
+WSL allows developers to run a Linux environment on Windows without the need for a separate virtual machine or dual booting. When using Zowe Explorer with WSL, two parallel processes take place: VS Code runs on the Windows operating system, while code execution and tooling happen within the Linux environment. 
+
+With Remote SSH, the network protocol provides users with a secure connection to a remote computer. When using this protocol with Zowe Explorer, you can securely connect to a remote machine or a remote development environment. 
+
+Both WSL and Remote SSH provide more tools for mainframe developers, but they also have limitations when it comes to credentials.
+
+Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, API Mediation Layer tokens &mdash; resides on the developer’s local machine, not the Linux environment or the remote server or virtual machine. This is because credential storage is managed by VS Code, which runs on Windows.
+
+Using the VS Code integrated terminal with virtual machines does not include access to the credentials stored by Zowe Explorer in the local machine. Zowe CLI, for example, is not able to retrieve credentials saved on a developer’s PC when accessing the mainframe. Instead it attempts to use any credentials stored in the virtual machine.
+
+This can lead to confusion and inconsistencies when authenticating and accessing resources.
+
+#### Solution
+
+It is crucial to ensure that credentials are carefully managed between the local machine and the remote server to maintain proper authentication. 
+
+To avoid any potential issues (such as being locked out) caused by credential mismatch or discrepancies, credentials in both local and virtual/remote machines must match.
```

Comment:
@zFernand0, I don't know that this is something we would want to mention here (to avoid confusing people), but it seems like there could be situations where a user may want to access services using different credentials in a VM or via ssh for testing purposes or to access different resources.

Would it be safer to say "credentials in both ... _should_ match."? It is acceptable as-is, but if there is nothing stopping a user who wants to intentionally use different credentials via VM or SSH, we may not want to tell them that they cannot do it.

Reply:
That's fair. 👍🏽 
We can be more lenient (._.?) in the way we state it 😋